README.md

```diff
@@ -23,9 +23,15 @@ There are many networking solutions available to suit a broad range of use-cases
  // Create a new controller instance
  controller := libnetwork.New()
  
- options := options.Generic{}
+ option := options.Generic{}
+ driver, err := controller.NewNetworkDriver("simplebridge", option)
```

Comment:
Can you please add a comment to the NewNetworkDriver call that, this is not required when the Plugins are made available.